releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,576 @@
+    - The `CREATE PLACEMENT POLICY` and `ALTER PLACEMENT POLICY` statements no longer support the `VOTERS` and `VOTER_CONSTRAINTS` placement options.
+    - TiDB ecosystem tools (TiDB Binlog, TiCDC, and BR) are now compatible with placement rules. The placement option is moved to a special comment in TiDB Binlog.
+    - The `information_schema.placement_rules` system table is renamed to `information_schema.placement_policies`. This table now only displays information about placement policies.
+    - The `placement_checks` system variable is replaced by `tidb_placement_mode`.
+    - It is prohibited to add partitions with placement rules to tables that have TiFlash replicas.
+    - Remove the `TIDB_DIRECT_PLACEMENT` column from the `INFORMATION_SCHEMA` table.
+- The `status` value of SQL plan management (SPM) binding is modified:
+    - Remove `using`.
+    - Add `enabled` (available) to replace `using`.
+    - Add `disabled` (unavailable).
+- DM modifies the OpenAPI interface
+    - Because of internal mechanism changes, the interface related to task management cannot be compatible with the previous experimental version. You need to refer to the new [DM OpenAPI documentation](/dm/dm-open-api.md) for adaptation.
+- DM changes the methods to resolve conflicts during the full import phase
+    - A `loader.&lt;name>.on-duplicate` parameter is added. The default value is `replace`, which means using the new data to replace the existing data. If you want to keep the previous behavior, you can set the value to `error`. This parameter only controls the behavior during the full import phase.
+- In v5.4 (v5.4 only), TiDB allows incorrect values for some noop system variables. Since v6.0.0, TiDB disallows setting incorrect values for system variables.
+
+## New features
+
+### SQL
+
+- SQL-based placement rules for data
+
+    TiDB is a distributed database with excellent scalability. Usually, data is deployed across multiple servers or even multiple data centers. Therefore, data scheduling management is one of the most important basic capabilities of TiDB. In most cases, customers do not need to care about how to schedule and manage data. However, with the increasing application complexity, deployment changes caused by isolation and access latency have become new challenges for TiDB. Since v6.0.0, TiDB officially provides data scheduling and management capabilities based on SQL interfaces. It supports flexible scheduling and management in dimensions such as replica counts, role types, and placement locations for any data. TiDB also supports more flexible management for data placement in multi-service shared clusters and cross-AZ deployments.
+
+    [User document](/placement-rules-in-sql.md)
+
+- Support building TiFlash replicas by databases. To add TiFlash replicas for all tables in a database, you only need to use a single SQL statement, which greatly saves operation and maintenance costs.
+
+    [User document](/tiflash/use-tiflash.md#按库构建-tiflash-副本)
```

Comment:
@shichun-0415 Please help update this anchor link when it is available. Thanks!

Reply:
> @shichun-0415 Please help update this anchor link when it is available. Thanks!

Fixed.